### ISSUE TITLE
[Profile Showcase] Prepare base models data for assets according to needs

### ISSUE THREAD

**Issue by noeliaSD** [status-im/status-desktop#13679]
### Description

1. Order in hidden section must always be the following (also after moving from in showcase to hidden, the order should be rearranged to the original order):
   - The current order in All accounts (so it needs to take into account the tokens management section).

2. There are also some restrictions and not all the assets in the model should be displayed:
[Design](https://www.figma.com/file/ibJOTPlNtIxESwS96vJb06/%F0%9F%91%A4-Profile-%7C-Desktop?type=design&node-id=2926%3A247232&mode=design&t=EJBavoiUCLIu1BS9-1)

![Screenshot 2024-02-22 at 12 11 44](https://github.com/status-im/status-desktop/assets/97019400/798f2ef4-3846-469c-b34b-17f2e382d35f)

**Comment by iurimatias** [comment]
moving to next milestone due to lack of space in this one

**Comment by noeliaSD** [comment]
> > moving to next milestone due to lack of space in this one
> 
> This is a must for the current milestone, if we are to deliver the Profile showcase feature right @alexjba @iurimatias

**Assets** can be moved to the next milestone, but I'd keep this task as low priority for this milestone in case we have the time.

**Comment by noeliaSD** [comment]
Blocked by https://github.com/status-im/status-desktop/issues/13492

**Comment by noeliaSD** [comment]
Removed from **2.29** milestone since `backend` team is blocked with their [task](https://github.com/status-im/status-desktop/pull/14291) bc of `wallet team` needs. **Wallet team** have now other priorities.
